js/ui/viewSelector.js

```diff
@@ -416,6 +476,8 @@ const ViewsClone = new Lang.Class({
                                                           enabled: false });
         iconGridClone.add_effect(this._saturation);
 
+        if (discoveryFeed !== null)
+            this.add_child(discoveryFeed);
```

Comment:
I actually tried that, but it has to happen after the constructing of the ViewsDisplayLayout.